docs/lakehouse-iceberg-catalog.md

```diff
@@ -264,16 +264,24 @@ You can pass [Iceberg table properties](https://iceberg.apache.org/docs/1.5.2/co
 
 The Gravitino server doesn't allow passing the following reserved fields.
 
-| Configuration item              | Description                                             |
-|---------------------------------|---------------------------------------------------------|
-| `comment`                       | The table comment.                                      |
-| `creator`                       | The table creator.                                      |
-| `location`                      | Iceberg location for table storage.                     |
-| `current-snapshot-id`           | The snapshot represents the current state of the table. |
-| `cherry-pick-snapshot-id`       | Selecting a specific snapshot in a merge operation.     |
-| `sort-order`                    | Selecting a specific snapshot in a merge operation.     |
-| `identifier-fields`             | The identifier fields for defining the table.           |
-| `write.distribution-mode`       | Defines distribution of write data                      |
+| Configuration item        | Description                                             | Default value | Required | Since Version |
+|---------------------------|---------------------------------------------------------|---------------|----------|---------------|
+| `comment`                 | The table comment.                                      | None          | No       | 0.2.0         |
+| `creator`                 | The table creator.                                      | None          | No       | 0.2.0         |
+| `current-snapshot-id`     | The snapshot represents the current state of the table. | None          | No       | 0.2.0         |
+| `cherry-pick-snapshot-id` | Selecting a specific snapshot in a merge operation.     | None          | No       | 0.2.0         |
+| `sort-order`              | Selecting a specific snapshot in a merge operation.     | None          | No       | 0.2.0         |
+| `identifier-fields`       | The identifier fields for defining the table.           | None          | No       | 0.2.0         |
+| `write.distribution-mode` | Defines distribution of write data                      | None          | No       | 0.2.0         |
+
+Gravitino server doesn't allow to change such properties:
+
+| Configuration item | Description                                  | Default value | Required | Since Version |
+|--------------------|----------------------------------------------|---------------|----------|---------------|
+| `location`         | Iceberg location for table storage.          | None          | No       | 0.2.0         |
+| `format`           | Iceberg table format like `iceberg/parquet`. | None          | No       | 0.2.0         |
+| `format-version`   | Iceberg meta data format version.            | None          | No       | 0.2.0         |
+| `provider`         | Table provider like `iceberg`.               | None          | No       | 0.2.0         |
```

Comment:
`provider` is the concept of Spark. I'm not sure whether it should be a property of Iceberg catalogs. Could you give more resources about it.

Reply:
Actually format version can be changed. v1 can change to v2. But v2 can't change to v1.

Reply:
We can change format, too. Iceberg can mix to use multiple formats. 
For example, partition 1 use parquet. partiton 2 use orc.

Reply:
location can be changed too. For one Iceberg table, they can have different locations.

Reply:
yes, they can changed by Iceberg, but we doesn't test and verify the action, we'd better keep it.

Reply:
Do we have a consensus on the changes?

Reply:
Users can change underlying Iceberg catalog these properties, you can't make them immutable. It will make users confusing.